responsive/best-design-quality

alaasaedy opened the pull request:
adding grid  cols to fit different screens.
remove unnecessary inline styling .
make qoute icon display in large screens only.
adding mx-auto to img column to center it in small screen

alaasaedy committed:
adding grid  cols to fit different screens.
remove unnecessary inline styling .
make qoute icon display in large screens only.
adding mx-auto to img column to center it in small screen

MAlngaawy reviewed (changes requested):
it's not responsive at all !
please look to it again carefully

alaasaedy committed:
Merge remote-tracking branch 'origin' into responsive/best-design-quality

alaasaedy committed:
destructure necessary data fetched from API's.
remove whole section, sub imgs static width, height and unnecessary styling.
transform right section img to normal img item instead of background and change sub imgs container to grid.

alaasaedy committed:
add section bg color

MAlngaawy reviewed (approved):
LGTM
